data/covering_grammar/tsv/ice_latn_broad.tsv

```diff
@@ -1,140 +1,123 @@
-a	aː
-a	a
-a	au
-a	ai
-a	aiː
-á	auː
-æ	ai
-æ	aiː
-au	øy
-au	øyː
-b	p
-b	pː
-bb	p
-bb	pː
-c	iː
-c	s
-c	kʰ
-c	h
-d	tː
-d	t
-ð	ð
-dd	tː
-e	ɛ
-e	ɛː
-e	ei
-é	jɛː
-é	jɛ
-ei	ei
-ei	eiː
-ey	ei
-ey	eiː
-f	fː
-f	f
-f	p
-f	v
-fnd	mt
-fnt	m̥t
-g	k
-g	cː
-g	kː
-g	c
-g	ɣ
-g	j
-g	x
-gg	kː
-gg	cː
-gg	k
-gj	c
-h	h
-hj	ç
-hl	l̥
-hn	n̥
-hr	r̥
-hv	kʰv
-i	i
-i	ɪ
-i	ɪː
-í	i
-í	iː
-j	j
-k	kʰ
-k	k
-k	cʰ
-k	c
-k	x
-kj	cʰ
-kj	c
-kk	c
-kk	hc
-kk	hk
-kk	x
-kl	hk
-kn	hk
-l	l
-l	lː
-l	l̥
-ll	tl
-ll	l:
-llt	l̥t
-lls	l̥s
-m	m
-m	mː
-m	m̥
-mm	m
-mm	mː
-n	n
-n	n̥
-n	nː
-ng	ŋk
-ng	ɲc
-ng	ŋ
-nk	ŋ̊k
-nk	ɲ̊c
-nn	n
-nn	nː
-nn	tn
-o	ɔː
-o	ɔ
-ó	ou
-ó	ouː
-ö	œ
-ö	œː
-p	p
-p	f
-p	pʰ
-pp	hp
-q	kʰ
-r	r̥
-r	rː
-r	r
-rl	rtl
-rl	rtl̥ 
-rn	rtn
-rnd	rnt
-s	s
-s	sː
-sl	stl
-sn	stn
-t	t
-t	tʰ
-t	ht
-þ	θ
-tt	ht
-u	ʏ
-u	ʏː
-u	u
-ú	uː
-ú	u
-v	v
-w	v
-x	s
-x	ks
-x	xs
-y	i
-y	ɪ
-y	ɪː
-ý	i
-ý	iː
-z	s
-
+a	aː
+a	a
+á	auː
+á	au
+æ	ai
+æ	aiː
+au	øy
+au	øyː
+b	p
+d	tː
+d	t
+ð	ð
+dd	tː
+e	ɛ
+e	ɛː
+e	e
+é	jɛː
+é	jɛ
+ei	ei
+ei	eiː
+ey	ei
+ey	eiː
+f	f
+ff	f
+ff	fː
+f	p
+f	v
+g	k
+g	cː
+g	kː
+g	c
+g	ɣ
+g	j
+g	x
+gg	kː
+gg	cː
+gg	k
+gj	c
+h	h
+h
+hj	ç
+hl	l̥
+hn	n̥
+i	i
+i	ɪ
+i	ɪː
+í	i
+í	iː
+j	j
+j
+j	c
+k	kʰ
+k	k
+k	cʰ
+k	c
+kj	cʰ
+kj	c
+kk	c
+kk	hc
+kk	hk
+kl	kʰ
+l	l
+l	lː
+l	l̥
+ll	tl
+ll	tl̥
+ll	l:
+m	m
+m	mː
+m	m̥
+mm	m
+mm	mː
+n	n
+n	n̥
+n	nː
+ng	ŋk
+ng	ŋc
+ng	ŋ
+nk	ŋ̊k
+nn	n
+nn	nː
+nn	tn
+o	ɔː
+o	ɔ
+ó	ou
+ó	ouː
+ö	œ
+ö	œː
+p	p
+p	pʰ
+pp	hp
+r	r̥
+r	rː
+r	r
+rl	rtl
+rl	rtl̥ 
+rn	rtn
+s	s
+s	sː
+ss	sː
+sl	stl
+sn	stn
+t	t
+t	tʰ
+t	ht
+þ	θ
+tt	ht
+u	ʏ
+u	ʏː
+u	u
+ú	uː
+ú	u
+v	v
+x	s
+x	ks
+x	xs
+y	i
+y	ɪ
+y	ɪː
+ý	i
+ý	iː
+	 
```

Comment:
I don't know for sure but I think two blank lines is different than (just) one.

Reply:
Hmm, interesting. It's not making any difference in the result though.